giza_actions/model.py

```diff
@@ -199,6 +276,35 @@ def _format_inputs_for_cairo(
 
         return {"job_size": job_size, "args": serialized}
 
+    def _format_inputs_for_ezkl(
+        self, input_file: str, input_feed: Dict, job_size: str, *args, **kwargs
```

Comment:
Maybe `input_file_path` or something here to specify it's a path? thought this paramter was the raw json

Reply:
It's following the same convention as `Cairo` this could be csv, numpy array and others, so functionality stays "the same" for different frameworks